src/System.Net.WebSockets.Client/src/System/Net/WebSockets/ClientWebSocket.cs

```diff
@@ -156,7 +156,7 @@ public Task ConnectAsync(Uri uri, CancellationToken cancellationToken)
 
         private Task ConnectAsyncCore(Uri uri, CancellationToken cancellationToken)
         {
-            _innerWebSocket = new WinHttpWebSocket();
+            _innerWebSocket = WebSocketHandle.Create();
 
```

Comment:
This PR will conflict with some code already checked in via TFS and is waiting on the existing tfs-git sync PR to be merged before it shows up. So the issue needs to be addressed whether it was in scope for this PR or not.